pipeline/dags/parallel_batch_download.py

```diff
@@ -81,34 +82,34 @@ def download_binding_batch(batch):
 
                     for dc_identifier in batch:
                         binding_id = utils.binding_id_from_dc(dc_identifier)
+                        base_path = os.path.join(BASE_PATH, utils.binding_download_location(binding_id, set_id))
+                        tmp_base_path = os.path.join(TMPDIR, utils.binding_download_location(binding_id, set_id))
```

Comment:
One more tiny thing that I could have come to think about on the first round but didn't: now we have `base_path` that is no longer a base path for the whole download like `BASE_PATH` is, but a more specific base path for a single binding. It might make us less likely to make stupid mistakes if we had `BASE_PATH` as we have now, and `base_path` and `tmp_base_path` were renamed to something like `binding_path` and `tmp_binding_path` or something similar that let's the reader know what this directory really is for.

This would need to be reflected in operator argument names too, but I suppose that's trivial in vscode.

Reply:
Done